v2/cmd/wails/internal/commands/build/build.go

```diff
@@ -269,14 +279,7 @@ func AddBuildSubcommand(app *clir.Cli, w io.Writer) {
 			// Calculate platform and arch
 			platformSplit := strings.Split(platform, "/")
 			buildOptions.Platform = platformSplit[0]
-			buildOptions.Arch = runtime.GOARCH
-			if system.IsAppleSilicon {
-				buildOptions.Arch = "arm64"
-			}
-			if len(platformSplit) == 2 {
-				buildOptions.Arch = platformSplit[1]
-			}
-
+			buildOptions.Arch = platformSplit[1]
```

Comment:
We should still initialise `buildOptions.Arch = defaultArch` and only set the second array item if it exists. Otherwise people might see a `index out of range` if they specify a platform without arch.

Reply:
Fixed. Copilot is freaky accurate...

Reply:
Never used it so far.